WordPress/src/main/res/values/dimens.xml

```diff
@@ -540,12 +540,20 @@
     <!-- Modal Layout Picker Styles -->
     <dimen name="mlp_header_scroll_snap_threshold">24dp</dimen>
     <dimen name="mlp_layouts_handle_margin_bottom">8dp</dimen>
-    <dimen name="mlp_layouts_row_height">150dp</dimen>
-    <dimen name="mlp_layouts_title_start_margin">15dp</dimen>
+    <dimen name="mlp_layouts_row_height">281dp</dimen>
     <dimen name="mlp_bottom_shadow_height">5dp</dimen>
     <dimen name="mlp_bottom_shadow_height_overlap">-5dp</dimen>
     <dimen name="mlp_bottom_button_vertical_margin">16dp</dimen>
     <dimen name="mlp_bottom_button_horizontal_margin">20dp</dimen>
+    <dimen name="mlp_bottom_button_horizontal_central_margin">16dp</dimen>
     <dimen name="mlp_categories_row_height">52dp</dimen>
-
+    <dimen name="mlp_layout_card_height">204dp</dimen>
+    <dimen name="mlp_layout_card_width">136dp</dimen>
+    <dimen name="mlp_layout_card_margin_start">20dp</dimen>
```

Comment:
We might want to consider referencing the existing margins. Also margin "29dp" seems a bit odd, is that intentional?

Reply:
I agree with you @malinajirka. I'm changing this to reference existing margins (`@dimen/margin_medium...large etc`) where possible.
I got the 29dp margin from Figma, but you are right. I made a mistake since I included the separator line. The correct value is 28dp!
|margin with the seperator line|margin without the separator|
|---|---|
|![Screenshot 2020-08-27 at 3 18 45 PM](https://user-images.githubusercontent.com/304044/91441445-e261c800-e878-11ea-9950-e52f87c22fbe.png)|![Screenshot 2020-08-27 at 3 38 30 PM](https://user-images.githubusercontent.com/304044/91443109-88163680-e87b-11ea-8aef-49ab3793e72a.png)|